packages/babel-parser/test/fixtures/comments/basic/call-expression-no-argument/input.js

```diff
@@ -0,0 +1 @@
+foo /* 1 */ (/* 2 */)
```

Comment:
The comment `/* 2 */` is now an `innerComment` of `ExpressionStatement`, which is not bad since 1) it is lost on current main and 2) we don't have AST structures for the bracket token.